src/test/unit/os/EnvTest.cpp

```diff
@@ -43,14 +40,14 @@ void EnvTest::RunTests()
   // get value that is not set
   value = "DONTCHANGEME";
   result = Env::get("NOTDEFINED", value);
-  TESTEQUAL2("get not set result", false, result);
-  TESTEQUAL2_STR("get not set value", "DONTCHANGEME", value.c_str());
+  ASSERT_FALSE(result);
+  EXPECT_STREQ("DONTCHANGEME", value.c_str());
```

Comment:
If an expect fails then you really have no idea what will happen when you continue execution since things are not working correctly. So I find it easier to just use assertions. But I suppose you are right that that might not be considered best practice. Fine as is.